care/facility/api/viewsets/patient.py

```diff
@@ -343,9 +343,17 @@ def discharge_patient(self, request, *args, **kwargs):
         last_consultation = PatientConsultation.objects.filter(patient=patient).order_by("-id").first()
         current_time = localtime(now())
         if last_consultation:
+            reason = request.data.get("discharge_reason")
+            notes = request.data.get("discharge_notes", "")
+            if not reason:
+                raise serializers.ValidationError(
+                    {"discharge_reason": "discharge reason is mandatory"}
+                )
+            last_consultation.discharge_reason = reason
```

Comment:
Shouldn't we perform some validation to ensure that the value is within the choices we specified?